testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/model/StoreProvider.java

```diff
@@ -31,25 +31,39 @@ public enum StoreProvider {
     JPA("jpa") {
         @Override
         public void addStoreOptions(List<String> commands) {
-            getDbVendor().ifPresent(vendor -> commands.add("--db=" + vendor));
+            getDbVendor().ifPresent(vendor -> {
```

Comment:
This switch seems to appear twice, so I wonder if it can be extracted.

The refactoring to a switch delivers IMHO the correct results, but switch statements without a `break` after each case are difficult to maintain, so I'd rather avoid them.

Reply:
Updated.

Reply:
Thanks. Let me know once there is a green build in your branch and then I'll review the PR again.